js/main_pg_preload.js

```diff
@@ -1,29 +1,7 @@
 /* 
-
 Javascript for Preloader Animation and Page navigation
-
 */
 
+import { preloadWithInterval } from "./modules/utils";
 
-function delay(url) {
-    window.setTimeout( () =>{
-    	window.location = url 
-    }, 1200);
-}

```

Comment:
The function `delay` was defined but never used anywhere. It's recommended to remove any unused code.

Reply:
The delay function is used in the .html  to delay the navigation page transitions.